src/main/java/org/eclipse/xpanse/terraform/boot/models/request/TerraformDeployWithScriptsRequest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.terraform.boot.models.request;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+/**
+ * Terraform uses the request object deployed by the script.
+ */
+@Data
+public class TerraformDeployWithScriptsRequest extends TerraformDeployFromDirectoryRequest {
+
+    @NotNull
+    @Schema(description = "List of script files for deployment requests deployed via scripts")
+    private String files;
```

Comment:
this must be a list of files . `List<String>` and simply each entry in this list is a file.

Reply:
@swaroopar I think this is a Map<String, Object> structure
![image](https://github.com/eclipse-xpanse/terraform-boot/assets/122504203/ddf1818c-e102-47f9-9d61-d5e4a69a030d)

Reply:
@WangLiNaruto ah ok. I see where is the confusion coming from. 
the parent `{}` is the TerraformDeployWithScriptsRequest. Not the scripts itself. `scripts` itself is a new field in `TerraformDeployWithScriptsRequest`. 
Create a new data model - `Script`.. with a one filed - content - string (we just need one field for now. but we need this to extend it later in case of new fields required)
then `scripts` must be of type -> List<Script>
and in TerraformDestroyWithScriptsRequest -> two new fields - scripts and tfstate. 
scripts -> List<Script>
tfstate -> string